tests/scipy_stats_test.py

```diff
@@ -49,6 +49,39 @@ def genNamedParametersNArgs(n):
 class LaxBackedScipyStatsTests(jtu.JaxTestCase):
   """Tests for LAX-backed scipy.stats implementations"""
 
+  @genNamedParametersNArgs(2)
+  def testBernoulliCdf(self, shapes, dtypes):
+    rng = jtu.rand_default(self.rng())
+    scipy_fun = osp_stats.bernoulli.cdf
+    lax_fun = lsp_stats.bernoulli.cdf
+
+    def args_maker():
+      x, logit = map(rng, shapes, dtypes)
+      x_dtype, p_dtype = x.dtype, logit.dtype
+      x = np.floor(x).astype(x_dtype)
+      p = expit(logit).astype(p_dtype)
+      return [x, p]
+
+    self._CheckAgainstNumpy(scipy_fun, lax_fun, args_maker, check_dtypes=False,
```

Comment:
We need to use the context `with jtu.strict_promotion_if_dtypes_match(dtypes):` here or this will fail in the X64 tests (see other examples in this file). Same below.